core/src/main/java/com/linecorp/armeria/internal/common/metric/RequestMetricSupport.java

```diff
@@ -343,6 +352,18 @@ public Counter writeTimeouts() {
         public Counter responseTimeouts() {
             return responseTimeouts;
         }
+
+        @Override
+        public DistributionSummary successAttempts() {
+            return parent.summary(idPrefix.name("actual.requests.attempts"),
+                                  idPrefix.tags("result", "success"));
```

Comment:
We should cache and use `newDistributionSummary`:
```suggestion
            if (successAttempts != null) {
                return successAttempts;
            }
            return successAttempts = newDistributionSummary(parent, idPrefix.name("actual.requests.attempts"),
                                                            idPrefix.tags("result", "success"));
```